xbmc/windows/GUIWindowDebugInfo.cpp

```diff
@@ -105,29 +105,29 @@ void CGUIWindowDebugInfo::Process(unsigned int currentTime, CDirtyRegionList &di
     StringUtils::ToLower(lcAppName);
 #if !defined(TARGET_POSIX)
     info = StringUtils::Format("LOG: {}{}.log\nMEM: {}/{} KB - FPS: {:2.1f} fps\nCPU: {}{}",
-                               CSpecialProtocol::TranslatePath("special://logpath").c_str(),
+                               CSpecialProtocol::TranslatePath("special://logpath"),
                                lcAppName.c_str(), stat.availPhys / 1024, stat.totalPhys / 1024,
```

Comment:
```suggestion
                               lcAppName, stat.availPhys / 1024, stat.totalPhys / 1024,
```

Reply:
you might also want to add a new line after MEM, same as for posix

Reply:
oh yes, missed this `c_str()`. nice catch.  but i think @lrusak wanted to take over in https://github.com/xbmc/xbmc/pull/19716 and there's a `\n` right after `{:2.1f} fps` for posix and non-posix

Reply:
> you might also want to add a new line after MEM, same as for posix

now i got it 😬😅